udata/tests/frontend/test_dataset_frontend.py

```diff
@@ -50,9 +51,28 @@ def test_render_list_empty(self):
 
     def test_render_display(self):
         '''It should render the dataset page'''
-        dataset = DatasetFactory()
-        response = self.get(url_for('datasets.show', dataset=dataset))
+        resource = ResourceFactory()
+        author = UserFactory()
+        dataset = DatasetFactory(tags=['foo', 'bar'],
+                                 resources=[resource],
+                                 description='a&éèëù$£',
+                                 owner=author)
+        url = url_for('datasets.show', dataset=dataset)
+        response = self.get(url)
         self.assert200(response)
+        json_ld = self.get_json_ld(response)
```

Comment:
It is not possible (or at least vey difficul) to put the tests at the model level. Indeed, we need url_for to generate the JSON/LD.